<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBack_bttn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/free-icon-media-player-5486140(1).png</normaloff>:/free-icon-media-player-5486140(1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="play_bttn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/free-icon-video-player-709696.png</normaloff>:/free-icon-video-player-709696.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_bttn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/27429.svg</normaloff>:/27429.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_bttn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/26445.png</normaloff>:/26445.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFront_bttn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/free-icon-media-player-5486140.png</normaloff>:/free-icon-media-player-5486140.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
selection-background-color: rgb(237, 212, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="volume_bttn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QTableView" name="playlistView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>661</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_bttn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/free-icon-add-8664564.png</normaloff>:/free-icon-add-8664564.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_bttn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/free-icon-sort-alphabet-ascending-81456.png</normaloff>:/free-icon-sort-alphabet-ascending-81456.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rand_bttn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/buttons.qrc">
      <normaloff>:/pngegg.png</normaloff>:/pngegg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string>Найти трек....</string>
    </property>
   </widget>
   <widget class="QListView" name="searchResultListView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_audio_file">
   <property name="text">
    <string>Open audio file</string>
   </property>
  </action>
  <action name="actionOpen_Audio_File">
   <property name="text">
    <string>Open Audio File</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images/buttons.qrc"/>
 </resources>
 <connections/>
</ui>
